### PR TITLE
test(services): add test for soundcloud oembed support

### PR DESCRIPTION
**Pull request by trieloff**
requires https://github.com/jacktuck/unfurl/pull/49

Please ensure your pull request adheres to the following guidelines:
- [ ] make sure to link the related issues in this description
- [ ] when merging / squashing, make sure the fixed issue references are visible in the commits, for easy compilation of release notes

## Related Issues

Thanks for contributing!

### CODE REVIEW

**Comment by rofe** [comment]
@trieloff the soundcloud test is working now, but the entire suite takes ~10 minutes (9 minutes idle time)...

**Comment by trieloff** [comment]
Strange that it's taking so long.

File: package.json [hunk 1]
```diff
@@ -42,6 +42,8 @@
     "@adobe/openwhisk-action-logger": "2.2.0",
     "@adobe/openwhisk-action-utils": "4.2.2",
     "@braintree/sanitize-url": "4.0.0",
+    "request": "2.88.0",
```
Comment (tripodsan) [inline package.json]: why are those added again?

File: test/embed.test.js [hunk 2]
```diff
@@ -20,16 +20,16 @@ const NodeHttpAdapter = require('@pollyjs/adapter-node-http');
...
-    keepAlive: false
+    keepAlive: false,
   },
   httpsProtocols: ['http1'],
   httpProtocols: ['http1'],
 }).fetch;
+const { assertContains } = require('./utils');
 
-const { embed } = proxyquire('../src/embed', { './unsplash': proxyquire('../src/unsplash.js', { '@adobe/helix-fetch': { fetch : (url) => testFetch(url) } })}); 
+const { embed } = proxyquire('../src/embed', { './unsplash': proxyquire('../src/unsplash.js', { '@adobe/helix-fetch': { fetch: (url) => testFetch(url) } }) });
```
Comment (tripodsan) [inline test/embed.test.js]: how did this ever pass lint?

**Comment by tripodsan** [comment]
> but the entire suite takes ~10 minutes (9 minutes idle time)...

I think the helix-fetch connections are not shut down correctly.... does it work correct locally?

File: package.json (hunk 1 above)
Comment (tripodsan) [reply package.json]: I removed them again....

File: test/embed.test.js (hunk 2 above)
Comment (tripodsan) [reply test/embed.test.js]: I was disabled!

**Comment by tripodsan** [comment]
> I think the helix-fetch connections are not shut down correctly.... does it work correct locally?

I think, I fixed it...although sometimes it still hangs...

**Comment by trieloff** [comment]
See also #171 – I don't think the issue has been introduced here, but in #165 

We haven't had a successful release since #165 has been merged, and I'd try with this PR. @rofe please merge when you have time.

File: test/embed.test.js (hunk 2 above)
Comment (rofe) [reply test/embed.test.js]: I was wondering the same thing 😁

**Comment by rofe** [comment]
This PR only had `test`, `chore` and `build` commits, which didn't trigger a semantic release 😉
